rosidl_generator_dotnet/resource/msg.cs.em

```diff
@@ -51,10 +66,15 @@ public class @(type_name) : IMessage {
             (NativeWriteField@(get_field_name(type_name, field.name))Type)Marshal.GetDelegateForFunctionPointer(
             native_write_field_@(field.name)_ptr, typeof(NativeWriteField@(get_field_name(type_name, field.name))Type));
 @[        else]@
-// TODO(esteve): Nested types are not supported
+
+        IntPtr native_get_field_@(field.name)_message_ptr = dllLoadUtils.GetProcAddress(nativelibrary, "native_get_field_@(field.name)_message");
+
+        @(type_name).native_get_field_@(field.name)_message = (NativeGetField@(get_field_name(type_name, field.name))MessageType)Marshal.GetDelegateForFunctionPointer(
```

Comment:
This could be rewritten as a getter, so that it is more idiomatic. Also, the naming convention may conflict with ROS's, so it'd be better named as for example `_HANDLE`

Reply:
I have changed _message by _HANDLE. 

I don't understand "This could be rewritten as a getter, so that it is more idiomatic". Please, explain this and/or give me an example